appveyor.yml

```diff
@@ -4,68 +4,85 @@ os: Visual Studio 2015
 
 environment:
   matrix:
-  - Toolset: v140
-  - Toolset: v120
-  - Toolset: v110
-  - Toolset: v100
+    - compiler: msvc-14-seh
```

Comment:
Looks like we are swtiching from matrix to actually spelling out which build are going to go
Why?
Cant we just one more toolset?  /*I am not very familiar with AppVeyor so forgive me if this is silly*/

Reply:
I refactored created by me `appveyor.yml` because of old file use `platform` variable, but it not useful for MinGW.

I just used my file from google/benchmark https://github.com/google/benchmark/commit/3336ea00d892fc312715c3a00d33a9568261e86a